repository_service_tuf_worker/repository.py

```diff
@@ -1208,6 +1221,116 @@ def bump_online_roles(self, force: Optional[bool] = False) -> bool:
 
         return True
 
+    def _run_force_online_metadata_update(self, roles: List[str]) -> List[str]:
```

Comment:
I'm wondering if the `metadata_update` is the best name here, as it confuses our process of updating the metadata.

Reply:
Well it is actually metadata update...
Hard to get a name correctly...